pyramid_oereb/views/webservice.py

```diff
@@ -452,14 +466,14 @@ def __parse_gnss__(self, gnss):
 
 
 class Parameter(object):
-    def __init__(self, flavour, format, with_geometry, images, identdn=None, number=None,
+    def __init__(self, format, flavour=None, with_geometry=False, images=False, identdn=None, number=None,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'format'](https://app.codacy.com/app/pyramid_oereb/pyramid_oereb/pullRequest?prid=4491046)